source/includes/connection-snippets/x509-snippets/csharp-connection.cs

```diff
@@ -0,0 +1,30 @@
+// begin x509 connection
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using System.Net.Security;
+
+using MongoDB.Bson;
+using MongoDB.Driver;
+using MongoDB.Driver.Core;
+using System;
+using System.Threading.Tasks;
+
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Cryptography;
+using System.Security.Permissions;
+
+var settingObjectOnlySettings = new MongoClientSettings {
```

Comment:
This object is missing its closing brace. (should come after the ``Server = ... `` line)

Reply:
Good catch, fixed!